api/db/changelog/db.changelog-219-add-id-me-access-module.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="eric" id="db.changelog-219-add-id-me-access-module">
+    <sql>
+      ALTER TABLE access_module
+        MODIFY COLUMN name
+        ENUM('ERA_COMMONS',
+        'TWO_FACTOR_AUTH',
+        'RAS_ID_ME',
+        'RAS_LOGIN_GOV',
+        'RT_COMPLIANCE_TRAINING',
+        'DATA_USER_CODE_OF_CONDUCT',
+        'PROFILE_CONFIRMATION',
+        'PUBLICATION_CONFIRMATION',
+        'CT_COMPLIANCE_TRAINING')
+    </sql>
+    <sql>
+      INSERT INTO access_module (name, expirable)
+      VALUES('RAS_ID_ME', false)
+    </sql>
+    <renameColumn newColumnName="ras_link_username"
```

Comment:
Share a single username for all RAS idp approaches, because a user should only be successfully using one of them.

Reply:
This looks dangerous  (but I am not sure how). 
During release process, we ran liquibase with old API binary reading the old name. 
But the risk might be low, because the rolling is quick and number of id verification is low. 
up to you.

Reply:
Good call. Thanks for pointing this out. What do you think of this updated approach?

Reply:
The best apporach in general should be:
1. Add new column, release to prod
2. Copy data from old column to new column. And start dual write in API
3. Verily columns are the same, then stop old column
4. Drop old column from DB
There are many releases required, and it is too complicated. Like I said above, id verification is rare, I think it should be ok to do what you are doing.